_posts/2017-12-08-alpha1.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Alpha, la 1ère formation à destination des acteurs de l’innovation du secteur public 
+authors: jihane.herizi
+---
+
+Après avoir [trouvé nos intrapreneurs de startup d’État](https://beta.gouv.fr/2017/02/27/comment-former-des-intrapreneurs.htmlÁ) nous vous expliquions, dans un précédent article, [comment ils sont quotidiennement formés en interne](https://beta.gouv.fr/2017/02/27/comment-former-des-intrapreneurs.html), notamment grâce à leurs coachs respectifs et aux autres membres des Startups d’Etat. 
+
+Pour aller plus loin, et parce que les Startups d’État et donc leurs intrapreneurs, vont être de plus en plus nombreux dans les mois et années à venir, il était nécessaire qu’une formation accélérée sur les méthodes de l’incubateur Beta.gouv. voit le jour. 
+<!--more-->
+
+C’est ainsi qu’Alpha est née le 7 novembre 2017 et a accueilli pendant 3 jours, dans les locaux de la Fabrique Pôle Emploi, 28 acteurs de l’innovation du secteur public, membres d’une Startup d’État ou désireux d’en créer une. 
+
+Avec eux, 14 intervenants coachs, intrapreneurs, développeurs, business développeurs venus les outiller sur des sujets tels que  « Comment trouver son premier client » à « Écouter ses utilisateurs partout le temps » en passant par « Survivre en milieu hostile » ou « Comment faire de l’industriel en mode artisanal ».
+
+![](/img/posts/Programme%20Alpha.png)
+
+[Cliquez ici pour (re)vivre en quelques tweets les temps forts de la formation !](https://storify.com/AlphaGouv/formation-alpha-premiere-session) 
```

Comment:
[Les liens en "Cliquez ici" sont à proscrire](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/)

Je propose : `[Revivez en quelques tweets les temps forts de la formation ](https://storify.com/AlphaGouv/formation-alpha-premiere-session)`

(j'ai également supprimé le point d'exclamation, surutilisé de nos jours)